README.md

```diff
@@ -336,6 +336,29 @@ function onInputKeyDown(event) {
 />
 ```
 
+### Supporting browser autofill
+
+Add the `autoComplete` prop to your select. Please notice that this feature is ignored for a Multiselect. The `autosize` prop should be set to `false` because autosize will shrink the input so the clickable area to trigger autofill suggestions does not expand over the whole select field.
+
+Usage example:
+```JS
+<Select
+	id="country"
+	multi={false} //required with autoComplete
+	autosize={false} //recommended with autoComplete
+	options={COUNTRIES}
+	simpleValue
+	name="country"
+	value={this.state.selectValue}
+	onChange={this.updateValue}
+	autoComplete="country"
+	onAutoFill={this.handleAutoFill}
+/>
+```
+
+You can use the `onAutoFill` callback to react to autofill state changes.
```

Comment:
What's the usecase for that?

Reply:
if you want to react to autofill state change on another level.. for example if you want to transport an `is-autofilled` to a parent node.

Reply:
`to autofill state changes, for example to adjust the styling of a wrapper component.`?